ambari-funtest/pom.xml

```diff
@@ -76,10 +88,15 @@
           <includes>
             <include>**/*.java</include>
           </includes>
-          <forkMode>once</forkMode>
         </configuration>
         <executions>
           <!-- Will display BUILD FAILURE if build fails or any test fails -->
+          <execution>
+            <id>integration-test</id>
+            <goals>
+              <goal>integration-test</goal>
```

Comment:
How does this work without adding the failsafe plugin to this project?

Reply:
It's already there.

Reply:
Oh, OK then! :)